infrastructure/proto/log_replication_cluster_info.proto

```diff
@@ -21,7 +21,7 @@ message ClusterConfigurationMsg {
 
 // Node configuration metadata
 message NodeConfigurationMsg {
-  string uuid = 1;
+  string connection_id = 1;
```

Comment:
what does connection id mean?

Reply:
connection id is the identifier for the transportation layer's connection. like IP for Netty and APH ID for APH